version_control/katalyst_by_codurance/templates/layouts/base.html

```diff
@@ -11,6 +11,7 @@
     {{ require_css(get_asset_url("../../css/main.css")) }}
     {{ require_css(get_asset_url("../../css/theme-overrides.css")) }}
     {{ standard_header_includes }}
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
```

Comment:
I wonder if instead of importing on the base style, we could do it only on the module that is being used. Also the fact the we are downloading the whole library it might affect to performance later on.

Reply:
Good idea! I will move the library inclusion to the module then